xcode/Subconscious/Shared/Services/UserProfileService.swift

```diff
@@ -292,6 +293,8 @@ actor UserProfileService {
             slugs: notes,
             sphere: self.noosphere
         )
+        let topEntries = entries
```

Comment:
Note that top entries ought to be determined by # incoming backlinks, while recent by modified.  These two queries are both a filter and a sort, so should be separate sql queries, rather than the same query sorted different ways.

Reply:
As discussed on our call, I'll put up a PR to hide this tab or just say "coming soon" when you view it for now instead of presenting the same data twice.